packages/gatsby-image/README.md

```diff
@@ -274,6 +280,8 @@ prop. e.g. `<Img fluid={fluid} />`
 | `className`        | `string` / `object` | Passed to the wrapper element. Object is needed to support Glamor's css prop                                                |
 | `style`            | `object`            | Spread into the default styles of the wrapper element                                                                       |
 | `imgStyle`         | `object`            | Spread into the default styles of the actual `img` element                                                                  |
+| `objectFit`        | `string`            | Spread into the default styles of the actual `img` element in a way that polyfills the behavior for IE. Defaults to `cover`. |
+| `objectPosition`   | `string`            | Spread into the default styles of the actual `img` element in a way that polyfills the behavior for IE. Defaults to `50% 50%`. |
```

Comment:
Do we need to mention IE here? having those options actually allow non-IE browsers to set those css properties, correct?

Reply:
Totally! I’ll remove the specific references to IE.